### PR TITLE
[5209] [studio][studio-ui] Lock the items currently in XB edit mode

### PR DESCRIPTION
**Pull request by dejan-brkic**
Update API docs

### Ticket reference or full description of what's in the PR
https://github.com/craftercms/craftercms/issues/5209

### CODE REVIEW

File: src/main/api/2/studio-api-2.yaml [hunk 1]
```diff
@@ -4249,6 +4249,87 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /content/item_lock_by_path:
```
Comment (sumerjabri) [inline src/main/api/2/studio-api-2.yaml]: Should it be plural, `items_lock*`?

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: src/main/api/2/studio-api-2.yaml (hunk 1 above)
Comment (dejan-brkic) [reply src/main/api/2/studio-api-2.yaml]: Requirements:
``` 
[BACKEND] Create API spec (OAS) to manage locks a set of items by path and review design
 item_lock_by_path
```

Comment (sumerjabri) [reply src/main/api/2/studio-api-2.yaml]: switch to items

**Review by sumerjabri — CHANGES_REQUESTED** [review]
update per convo/notes

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**